main.py

```diff
@@ -31,27 +31,23 @@ async def on_message(message):
 	global category
 	global userMessage
 
+	# if the message is not a dm or if message is from bot, return (Do not proceed)
+	if (not isinstance(message.channel, discord.channel.DMChannel) or message.author == client.user):
+		return
+	
```

Comment:
Ignore any messages thats sent by the bot OR not in DM. So every if function afterwards does not need to include it.